lib/media/track/localvideotrack.js

```diff
@@ -39,6 +39,9 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
     super(mediaStreamTrack, options);
 
     Object.defineProperties(this, {
+      _unprocessedTrack: {
```

Comment:
Why do we need this? Doesn't `this.mediaStreamTrack` point to the unprocessed track?

Reply:
Update mediatrack.js to save the unprocesstrack